src/Commands/Scheduled/LinkChecker.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Portable\FilaCms\Commands\Scheduled;
+
+use Illuminate\Console\Command;
+use Portable\FilaCms\FilaCms;
+use Portable\FilaCms\Models\LinkCheck;
+use Str;
+use Portable\FilaCms\Jobs\CheckLink;
+
+class LinkChecker extends Command
+{
+    protected $signature = 'fila-cms:verify-links';
+
+    protected $description = 'Install Fila CMS';
```

Comment:
You might want to update this to give a more accurate description

Reply:
yep my bad on this